src/SonarScanner.MSBuild.TFS/BuildVNextCoverageReportProcessor.cs

```diff
@@ -39,6 +39,9 @@ public BuildVNextCoverageReportProcessor(ICoverageReportConverter converter, ILo
         this.directoryWrapper = directoryWrapper ?? DirectoryWrapper.Instance;
     }
 
+    // The logic in this method is flawed:
+    // https://sonarsource.atlassian.net/browse/SCAN4NET-786
+    // https://sonarsource.atlassian.net/browse/SCAN4NET-787
```

Comment:
As we have tickets, I'd make these ToDos with more clear notes.

```suggestion
    // ToDo: SCAN4NET-786 Test report discovery is flawed
    // ToDo: SCAN4NET-787 Coverage fallback should be in AzDo Extension
```